sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/SnapshotInfo.java

```diff
@@ -69,7 +70,7 @@ public Row toRow() {
     try {
       SchemaRegistry registry = SchemaRegistry.createDefault();
       CODER = registry.getSchemaCoder(SnapshotInfo.class);
-      SCHEMA = registry.getSchema(SnapshotInfo.class).sorted();
+      SCHEMA = registry.getSchema(SnapshotInfo.class).sorted().toSnakeCase();
```

Comment:
Another breaking change that I'm comfortable with. Better to unify everything to snake case format.